cmd/panfs-multipart.go

```diff
@@ -271,7 +271,12 @@ func (fs *PANFSObjects) NewMultipartUpload(ctx context.Context, bucket, object s
 		return nil, err
 	}
 
-	if err = ioutil.WriteFile(pathJoin(uploadIDDir, fs.metaJSONFile), fsMetaBytes, 0o666); err != nil {
+	filepath := pathJoin(uploadIDDir, fs.metaJSONFile)
+	if err = ioutil.WriteFile(filepath, fsMetaBytes, fs.defaultObjMode); err != nil {
+		logger.LogIf(ctx, err)
+		return nil, err
+	}
+	if err = os.Chown(filepath, fs.defaultOwner, fs.defaultGroup); err != nil {
```

Comment:
we, seems, handle upload metadata file, which we can keep root permissions for as well.

Reply:
Rolled back